development/image-builder/README.md

```diff
@@ -104,6 +104,17 @@ If the job is running in CI (Prow), it picks up the current `org/repo` value fro
 
 Currently, image-builder contains a basic implementation of a notary signer. If you want to add a new signer, refer to the [`sign`](./sign) package, and its code.
 
+## Named Tags
+
+image-builder supports passing name along with tag both via -tag option or config for tag template.
+You can use `-tag name=value` to pass name for tag. 
+
+If name is not provided, it will be evaluated from value:
+ - if value is string it will be used as name directly eg.: 
+   - `-tag latest` will be equal to `-tag latest=latest`
+ - if value is go-template it will be converted to valid name eg:
+   - `-tag v{{ .ShortSHA }}-{{ .Date }}` will be equal to `-tag vShortSHA-Date=v{{ .ShortSHA }}-{{ .Date }}`
```

Comment:
```suggestion
If the name is not provided, it is evaluated from the value:
 - if the value is a string, it is used as a name directly. For example,`-tag latest` is equal to `-tag latest=latest`
 - if the value is go-template, it will be converted to a valid name. For example, `-tag v{{ .ShortSHA }}-{{ .Date }}` is equal to `-tag vShortSHA-Date=v{{ .ShortSHA }}-{{ .Date }}`
```